src/main/java/alto/Featurizer.java

```diff
@@ -64,7 +64,7 @@ public void featurize(String featuresDir) throws IOException{
 	}
 
 	public void fillVocab() throws IOException{
-		FileInputStream infstream = new FileInputStream(this.baseDir+"results/"+util.Constants.CORPUS_NAME + "/input/"+util.Constants.CORPUS_NAME+ ".voc");
```

Comment:
was still reading in corpus name from Constants, instead of from properties file.